projects/clearpool/index.js

```diff
@@ -125,10 +126,32 @@ Object.keys(config).forEach((protocol) => {
     const borrowed = async (api) => {
       const { pools, tokens } = await _getLogs(api);
       const bals = await api.multiCall({ abi: borrowFn, calls: pools });
-      api.addTokens(tokens, bals);
+     return api.addTokens(tokens, bals);
     };
+    if (!module.exports[chain]) {
```

Comment:
@Addis-Belete The code is good, but we need a more readable version to be easily understood.

Reply:
let's use the `api` methods that they provide maybe we can somehow benefit from them